Cubed-Development/Modern-Warfare-Cubed
# Korean Translation

## 🤔 What type of PR is this? (check all applicable)

- [ ] 🍕 Addition
- [ ] ⌨️ Productivity
- [ ] 🌟 Quality Of Life

## 📝 Description

This PR adds Korean Translation.

## 🧐 The Rationale



Not every korean are fluent with english. this will greatly help players like me who wants to play modded SMP which includes MWC with native friends.

## 🎯 Key Objectives


Add the korean Translation to MWC.

## 🚦 Testing 


1. Start minecraft
2. Set language to Korean,
3. Join world and check MWC's item names are in korean

## ⏮️ Backwards Compatibility 



This PR only adds 1 Language file. I don't think there is anything to break here 🤷‍♂️ 


## 📚 Related Issues & Documents



N/A.

## 🖼️ Screenshots/Recordings

![2023-11-08_01 05 11](https://github.com/Cubed-Development/Modern-Warfare-Cubed/assets/49832212/9aec4ef8-828c-4445-864b-08934ad9662f)

![2023-11-08_01 04 45](https://github.com/Cubed-Development/Modern-Warfare-Cubed/assets/49832212/3def12e1-5ea8-49f3-9cd3-edb91e5f0707)

## 📖 Added to documentation?


- [ ] 📓 Javadoc

## 😄 [optional] What gif best describes this PR or how it makes you feel?
![cheers](https://media.tenor.com/IErQHBRt6GIAAAAd/leonardo-dicaprio.gif)
